files/gitconfig

```diff
@@ -6,4 +6,5 @@
 [merge]
     renames = false
 [core]
-    autocrlf = input
+    safecrlf = true
+    # autocrlf = input
```

Comment:
Removing `autocrlf` fixes the issue. I think I misunderstood the role of this config, and it's needed. I'll revert the commit which introduced it.

Thanks for looking into the issue.

Reply:
okay, makes sense to me